src/calculateRentalCost.js

```diff
@@ -20,7 +20,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return (days * 40);
+  } else if (days < 7) {
+    return (days * 40) - 20;
+  } else {
```

Comment:
you no need this else
```suggestion
  }
```